Resources/Audio/Ambience/Antag/license.txt

```diff
@@ -0,0 +1,5 @@
+﻿nukeops_start.ogg was taken from: https://github.com/tgstation/tgstation/commit/827967c9650c23af64280ad1491405fed8f644c5
+
+pirate_start.ogg was taken from: https://github.com/SerbiaStrong-220/space-station-14/pull/21/commits/15aa6e22a3914f80ce4537a710a1bb52ae87e27c
+
+zombie_start.ogg was taken from: https://github.com/SerbiaStrong-220/space-station-14/pull/21/commits/15aa6e22a3914f80ce4537a710a1bb52ae87e27c
```

Comment:
are these two already in the game? (iirc you said pirate_start is actually monkey1.ogg so i assume zombie_start is the same) if so please reference those files directly in the code instead of duplicating them, also referencing your own pr in another fork as the source for the sound won't be accepted unless you made it

Reply:
Those files are original files from another forked server. We will deal with it today.

Reply:
You should mention who made them then in the RGA (i took a look at the new RGA in this pr and it seems to mention different files?)

Reply:
Done, i was waiting for approval of a PR.